.cicd/test-package.anka.sh

```diff
@@ -10,4 +10,9 @@ if [[ ! $(brew --version 2>/dev/null) ]]; then
    perform 'brew analytics off'
```

Comment:
We should fix the `curl` piping to `bash` here, for the [same reasons](https://github.com/EOSIO/eos/pull/10555#discussion_r674339192).